app/models/manageiq/providers/container_manager.rb

```diff
@@ -30,5 +32,10 @@ def aggregate_cpu_total_cores(targets = nil)
     def aggregate_memory(targets = nil)
       aggregate_hardware(:computer_systems, :memory_mb, targets)
     end
+
+    def add_default_policies
+      # make sure that add_policy indeed adds a policy profile (aka MiqPolicySet)
+      add_policy(MiqPolicySet.find_by_guid('747a70d0-a8bf-4b63-bdad-1676bc0c5356'))
```

Comment:
> Can we prioritize getting the required PRs merged first before trying to review this? It's hard to keep track of what's going on if the parent PRs change.
> 
> Note, I'm wondering if the guid was because we need to seed separate classes and needed a way to link them up together... maybe we seed them together in a single class, we can avoid needing this guid entirely? I honestly don't know, I have a hard time following PRs that are more than 1 deep in terms of dependent PRs. I usually make a PR with dependencies as WIP and don't have others review until the required PRs are merged.

Sound like good advice, I will follow it from this point on.

@jrafanie @blomquisg @gtanzillo Feel free to remove yourselves from any [WIP] patches and I will re-add you when dependencies are met (I'm never quite sure how you work with PRs commits and dependencies so these comments are useful) I will continue the guid discussion on #7413, where I think it is more relevant.